CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Awesome React App
+
+### Introduction
+First of all, thank you for considering to contribute! 🙏
+
+Everybody can help make this project a little more awesome, doesn't matter if it's a big or small contribution.
+Please make sure you follow the code of conduct, which is located in the repository master directory, in order for everyone to have a peaceful and progressive experience.
+Find out below, which steps are needed to help out exactly.
+
+### How to contribute
+Follow these simple steps:
+1. Have a look at the issue page of the repository [here](https://github.com/ifactory-solutions/awesome-react-app/issues).
```

Comment:
Have a look at the [issue page](https://github.com/ifactory-solutions/awesome-react-app/issues) page of the repository 
Looks cleaner, what do you think?

Reply:
Definitely.